packages/core/src/node/bunyan-logger-server.ts

```diff
@@ -42,11 +42,12 @@ export class BunyanLoggerServer implements ILoggerServer {
         if (['trace', 'debug', 'info', 'warn', 'error', 'fatal'].indexOf(logLevel) < 0) {
             logLevel = 'info';
         }
+    }
 
-        this.loggers.push(bunyan.createLogger({
-            name: 'Theia',
-            level: logLevel
-        }));
+    create(serveroptions: object | undefined) {
```

Comment:
I don't think you need this method, but an option should be injected into the constructor of `BunyanLoggerServer`. This option should be bounded as a constant value in `loggerBackendModule'.